src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.DefaultItems.props

```diff
@@ -17,6 +17,31 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   <Import Project="$(NETCoreSdkBundledVersionsProps)" Condition="Exists('$(NETCoreSdkBundledVersionsProps)')" />
 
+  <ItemGroup Condition="'$(UseRefTargetingPacks)' == 'true'">
```

Comment:
There should be a bug referenced to remove this.

Reply:
Filed #2962 for this (not referenced in the code though).